benchmark/perf-tests/1000-raycluster/wait-for-rayclusters.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Both `1000-raycluster` and `5000-raycluster` directories have `wait-for-raycluster.sh`. Could we unify them?

Reply:
Yeah we could, can I do it in a follow-up when we introduce the 10K tests?